modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi

```diff
@@ -0,0 +1,60 @@
+@App:name('ReadFileInBinaryChunkedMode')
+
+@App:description('Read the file as binary chunks')
+
+/*
+Purpose:
+    This application demonstrates how to configure WSO2 Streaming Integrator Tooling to read the file and send events
```

Comment:
Rephrase as: "This application demonstrates how to configure WSO2 Streaming Integrator Tooling to read a binary file in chunks and send via WebSocket transport."

Reply:
fixed in https://github.com/wso2/streaming-integrator-tooling/pull/87/commits/3b0b69ece28668ca58eaee3041d4cb7636494296